### ISSUE TITLE
OSQP init_params

### ISSUE THREAD

**Issue by pharringtonp19** [google/jaxopt#179]
How should we initialize the parameters in this problem setting?

```
from jaxopt import OSQP

Q = 2 * jnp.array([[2.0, 0.5], [0.5, 1]])
c = jnp.array([1.0, 1.0])
A = jnp.array([[1.0, 1.0]])
b = jnp.array([1.0])
G = jnp.array([[-1.0, 0.0], [0.0, -1.0]])
h = jnp.array([0.0, 0.0])

qp = OSQP()
sol = qp.run(params_obj=(Q, c), params_eq=(A, b), params_ineq=(G, h)).params

print(sol.primal)
print(sol.dual_eq)
print(sol.dual_ineq)
```

`TypeError: run() missing 1 required positional argument: 'init_params'`

**Comment by pharringtonp19** [comment]
Solution: `init_params=None` as in the following

```
sol = qp.run(init_params=None, params_obj=(Q, c), params_eq=(A, b), params_ineq=(G, h)).params
```

**Comment by mblondel** [comment]
Hi @pharringtonp19! Thanks a lot for your question. Indeed, your code snippet shows that this was confusing. Thankfully, `init_params=None` is now the default behavior, since this [commit](https://github.com/google/jaxopt/commit/60935e40ee29ed91c7433b55827d801a3fe28efb), which happened after the 0.3 release. I'll make a minor release next week. So hopefully, after the release, you should be able to remove `init_params=None` from your code.

**Comment by mblondel** [comment]
I just released version 0.3.1 which should now set `init_params=None` by default. Please run `pip install jaxopt` :)
